src/jit/interface.h

```diff
@@ -0,0 +1,18 @@
+/* Function for getting effective (JIT/specialized/original) bytecode. */
+MVM_STATIC_INLINE MVMuint8 * MVM_frame_effective_bytecode(MVMFrame *f) {
```

Comment:
Why did this need to move from `frame.h`?

Reply:
Arguably it didn't. Maybe better not.
The thinking here was that the `interface.h` header would 'own' all JIT-related parts. That didn't totally work out anyway, so I'll be happy to put this back.

Reply:
Oh hang on. This used to live in `compile.h`, which is why it now lives in `interface.h`. But I can move it to `frame.h` instead.

Reply:
And now I know why I didn't do that. The MVMJitCode structure isn't yet defined at the point where we load `frame.h`. So I'll fix that up some other day.